.circleci/config.yml

```diff
@@ -543,7 +559,7 @@ jobs:
 workflows:
   version: 2.1
 
-  anvil-per-test:
+  skip:
```

Comment:
Is this name change on purpose?

Reply:
Yeah just don't want it to be this oddly specific